src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -190,7 +190,7 @@ protected override bool ProcessRequest(IRequestInfo info)
 				return true;
 			}
 
-			if (localPath.StartsWith(OriginalImageMarker))
+			if (localPath.StartsWith(OriginalImageMarker) && IsImageTypeThatCanBeDegraded(localPath))
```

Comment:
Seems you're trying to fix a problem where some CSS files got the OriginalImageMarker prefix stuck in front? Feels like the bug was there rather than here...why would we want or need to do this to CSS files? We don't need to protect them against image compression!
One effect of your change will (I think) be that any .svg file that is marked with OriginalImageMarker will not be found at all, since there is no special logic for searching in multiple places for .svg files, and they cannot be downgraded according to the new definition. Is that intentional (or at least irrelevant because we don't use them?)
Similarly anything ELSE that gets so marked typically won't be found at all...it's only CSS files, AFAIK, that ignore the path and search for the file. So we'd better be sure nothing else we want found is getting this marker by mistake...